### PR TITLE
docs: add package pronunciation

### PR DESCRIPTION
**Pull request by NozomuIkuta**
### 🔗 Linked issue

No related issues in this repository.

### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation, readme, or JSdoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

This PR is created based on https://github.com/unjs/website/issues/168#issuecomment-1791691182.
Pronunciation symbols are copied from Cambridge Dictionary:

- [h](https://dictionary.cambridge.org/dictionary/english-japanese/h)
- [three](https://dictionary.cambridge.org/dictionary/english-japanese/three)

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [x] I have updated the documentation accordingly.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,7 @@
 [![License][license-src]][license-href]
 [![JSDocs][jsdocs-src]][jsdocs-href]
 
-H3 is a minimal h(ttp) framework built for high performance and portability.
+H3 (pronounced as /eɪtʃθriː/, like h-3) is a minimal h(ttp) framework built for high performance and portability.
```
Comment (NozomuIkuta) [inline README.md:10]: Some packages' READMEs don't have introduction sentence like this: for example, [c12's README](https://github.com/unjs/c12) just says "Smart Configuration Loader.".

So, we might want to get all REAMEs to have an introduction sentence like "c12 (pronounced as /.../, like c-twelve) is a ...".

Comment (Barbapapazes) [reply README.md:10]: Yes!

**Review by Barbapapazes — APPROVED** [review]

**Comment by NozomuIkuta** [comment]
@pi0 

How does this PR look to you? 👀 
If you are fine, @Barbapapazes and I would be able to work on other repositories.

**Review by pi0 — APPROVED** [review]

**Comment by pi0** [comment]
Thanks!

**Pull request merged**